### PR TITLE
Add main field to package.json

### PR DESCRIPTION
**Pull request by ricardo-devis-agullo**
Hi,

This is just a simple PR to add the main field to the package.json, because Webpack gives me some problems when using [createHmac](https://github.com/crypto-browserify/createHmac) that depends on this one.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "sha.js",
   "description": "Streamable SHA hashes in pure javascript",
-  "version": "2.4.8",
+  "version": "2.4.9",
```
Comment (fanatid) [inline package.json]: please remove this, version will be changed by maintainer

**Comment by dominictarr** [comment]
`main` should default to `index.js` if not given, if webpack is having problems with that, it's a problem with webpack because many modules that use `index.js` will not have `main` set. This should be an issue on webpack not sha.js

**Comment by ricardo-devis-agullo** [comment]
Even if that were true, still doesn't do any harm to add it, doesn't it?

File: package.json (hunk 1 above)
Comment (ricardo-devis-agullo) [inline package.json]: done :)

**Comment by dominictarr** [comment]
well, it sort of does because it means people then have to go around making changes to every other module because of a bug in webpack, that wastes a lot more time than just fixing webpack

**Pull request closed without merging**